modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc

```diff
@@ -0,0 +1,253 @@
++
+.Example: Creating a `Canary service`
+[source,yaml]
+----
+# Canary traffic is directed to this Service, for example, it might initially be 30% of traffic
+apiVersion: v1
+kind: Service
+metadata:
+  name: argo-rollouts-canary-service #<1>
+spec:
+  ports:
+  - port: 80
+    targetPort: http
+    protocol: TCP
+    name: http
+
+  selector: # this field/sub-field must have the same value as Rollout
+    app: rollouts-demo
+    # Rollouts will automatically update the Service with pod-template-hash of the canary ReplicaSet. e.g.:
+    # rollouts-pod-template-hash: 7bf84f9696
+----
+<1> Name of the canary service created in the Create Services page
++
+[NOTE]
+====
+Ensure that the name of the canary service created in the *Create Route* page matches with the name of the canary service created in the *Create Services* page.
+====
+.. Click *Create* to create the canary service.
+.. Similarly, repeat these steps to create the stable service by adding the following snippet in the YAML view of the *Create Services* page:
++
+.Example: Creating a `Stable service`
+[source,yaml]
+----
+# Stable traffic is directed to this Service, for example, it might initially be 70% of traffic.
+apiVersion: v1
+kind: Service
+metadata:
+  name: argo-rollouts-stable-service #<1>
+spec:
+  ports:
+  - port: 80
+    targetPort: http
+    protocol: TCP
+    name: http
+
+  selector: # this field/sub-field must have the same value as Rollout 
+    app: rollouts-demo 
+    # Rollouts will update the Service with pod-template-hash of the stable (non-canary) ReplicaSet. e.g.:
+    # rollouts-pod-template-hash: 1b6a7733
+----
+<1> Name of the stable service created in the Services page
+.. Click *Create* to create the stable service.
++
+[NOTE]
+====
+Ensure that the name of the stable service created in the *Create Route* page matches with the name of the stable service created in the *Create Services* page.
+====
+. Create the `Rollout` CR to reference the Route and the Services.
+.. In the Administrator perspective of the web console, navigate to *Operators* -> *Installed Operators* -> *Red Hat OpenShift GitOps* -> *Rollout*.
+.. On the *Create Rollout* page, click *YAML view* and add the following snippet:
++
+.Example: Creating a `Rollout`
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: rollouts-demo
+spec:
+  template:
+    metadata:
+      labels:
+        app: rollouts-demo
+    spec:
+      containers:
+      - name: rollouts-demo
+        image: argoproj/rollouts-demo:blue
+        ports:
+        - name: http
+          containerPort: 8080
+          protocol: TCP
+        resources:
+          requests:
+            memory: 32Mi
+            cpu: 5m
+                
+  revisionHistoryLimit: 2
+  replicas: 5
+  strategy:
+    canary:
+      canaryService: argo-rollouts-canary-service # must match name of Service above #<1>
+      stableService: argo-rollouts-stable-service # must match name of Service above #<2>
```

Comment:
Same comment about the comments in YAML. Based on the answer, I will provide more suggestions to this example YAML file.

Reply:
Fixed as per earlier comment.